### DISCUSSION TITLE
cannot config route as it already presented (RTNETLINK answers: File exists)

### DISCUSSION THREAD

**Discussion by initChat** [srsran/srsRAN_Project#506] [General Help] [answered]
Hello, I have problem when following this tutorial https://docs.srsran.com/projects/project/en/latest/tutorials/source/srsUE/source/index.html#zeromq-based-setup. At step **Testing the Network** and **Routing Configuration**. **I cannot add route as it is already presented.**
```
sudo ip ro add 10.45.0.0/16 via 10.53.1.2
RTNETLINK answers: File exists
```
my route is below.
```
$ route -n
Kernel IP routing table
Destination     Gateway         Genmask         Flags Metric Ref    Use Iface
0.0.0.0         192.168.1.1     0.0.0.0         UG    600    0        0 wlp9s0
10.45.0.0       0.0.0.0         255.255.0.0     U     0      0        0 ogstun
10.53.1.0       0.0.0.0         255.255.255.0   U     0      0        0 br-2d2baf4f3a8c
169.254.0.0     0.0.0.0         255.255.0.0     U     1000   0        0 wlp9s0
172.17.0.0      0.0.0.0         255.255.0.0     U     0      0        0 docker0
172.19.1.0      0.0.0.0         255.255.255.0   U     0      0        0 br-36c6988d1fa9
192.168.1.0     0.0.0.0         255.255.255.0   U     600    0        0 wlp9s0
```
My config is open5gs on docker of srsRAN_Project, gNB from srsRAN_Project, and ue from srsRAN4G. This is zmq based with no actual radio or UE.
My system runs well, but I stuck at this step.

Thank you very much.

**Comment by pgawlowicz** [comment, marked as answer]
you probably have Open5gs installed from source, right? 
please check this section: https://docs.srsran.com/projects/project/en/latest/tutorials/source/srsUE/source/index.html#nd-open5gs-instance-installed-manually

**Reply by initChat** [reply]
Thank you it works. Thank you very much
